Microsoft.Recognizers.Text.sln

```diff
@@ -1,33 +1,51 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Recognizers.Text.Number", "Microsoft.Recognizers.Text.Number\Microsoft.Recognizers.Text.Number.csproj", "{61205449-BDBA-4BF3-8B24-AE059E0B1B61}"
-EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Recognizers.Text.Number.Tests", "Microsoft.Recognizers.Text.Number.Tests\Microsoft.Recognizers.Text.Number.Tests.csproj", "{7333C1ED-A0CC-4F2C-AEF4-B9DBC9ED988D}"
-EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tests", "Tests", "{ED7B6456-AB0A-48CE-8F85-711FE87F09C2}"
-EndProject
-Global
-	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
-		Release|Any CPU = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{61205449-BDBA-4BF3-8B24-AE059E0B1B61}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{61205449-BDBA-4BF3-8B24-AE059E0B1B61}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{61205449-BDBA-4BF3-8B24-AE059E0B1B61}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{61205449-BDBA-4BF3-8B24-AE059E0B1B61}.Release|Any CPU.Build.0 = Release|Any CPU
-		{7333C1ED-A0CC-4F2C-AEF4-B9DBC9ED988D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{7333C1ED-A0CC-4F2C-AEF4-B9DBC9ED988D}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{7333C1ED-A0CC-4F2C-AEF4-B9DBC9ED988D}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{7333C1ED-A0CC-4F2C-AEF4-B9DBC9ED988D}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(NestedProjects) = preSolution
-		{7333C1ED-A0CC-4F2C-AEF4-B9DBC9ED988D} = {ED7B6456-AB0A-48CE-8F85-711FE87F09C2}
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15

```

Comment:
Fine to leave it as 2017.  I'm guessing upgrading LUIS itself will be a little more involved, but if we integrate as nugget we should be able to use the 2017 version.